buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -388,6 +388,14 @@
       "value": "false",
       "allowOverride": true
     },
+    "PB_ConfigurationGroup": {
+      "value": "Debug",
+      "allowOverride": true
+    },
+    "PB_Platform": {
```

Comment:
It's the symbol publishing step, leading into indexing: https://github.com/chcosta/corefx/blob/c9f94af56ab8f68930672836404b59317eef1a51/buildpipeline/DotNet-CoreFx-Trusted-Windows.json#L244

Without this, it can't find any PDBs to put on symweb.